config/example/ir.yaml

```diff
@@ -19,6 +19,59 @@ morph:
   validators: # List of hex-encoded 33-byte public keys of sidechain validators to vote for at application startup
     - 0283120f4c8c1fc1d792af5063d2def9da5fddc90bc1384de7fcfdda33c3860170
   switch_interval: 2m # interval b/w RPC switch attempts if the node is not connected to the highest priority node
+  consensus: # Local consensus launch mode activated only when 'endpoint.client' is unset.
+    magic: 15405 # Network magic. Must be unsigned integer in range [1:4294967295]
+    committee: # Initial committee
+      - 02b3622bf4017bdfe317c58aed5f4c753f206b7db896046fa7d774bbc4bf7f8dc2 # Hex-encoded public key
+    storage: # Blockchain storage
+      type: boltdb # One of following storage types:
+        # boltdb (local BoltDB)
+        # leveldb (local LevelDB)
+        # inmemory (volatile storage, mostly suitable for testing)
+      #
+      # If 'path' exists, it should correspond to the selected variant
+      path: ./db/morph.bolt # File system path to the storage. Ignored for 'inmemory' storage.
+    notary_disabled: true # Optional flag to disable Notary service. Defaults to false
+    time_per_block: 1s # Optional time period (approximate) between two adjacent blocks. Defaults to 15s.
+      # Must not be negative
+    max_traceable_blocks: 2102400 # Optional length of the chain accessible to smart contracts. Defaults to 2102400.
+      # Must not be greater than 4294967295
+    seed_nodes: # Optional list of existing nodes to communicate with over Neo P2P protocol. By default, node runs as standalone
+      # Same format as 'p2p.listen'
+      - node2:20333
+      - node3:20333
+    hardforks: # Optional hard-forks
+      name: 1730000 # Maps name to chain height. Heights must not be greater than 4294967295
+    validators_history: # Optional number of consensus nodes to use after given height
+      10: 7 # Maps chain height to number of consensus nodes. Values must not be greater than 2147483647
+    native_activations: # Optional chronology of native contracts updates
```

Comment:
This setting is treated as follows: if no contracts are specified, then the default value `[0]` will be applied for all known native contracts. If *some* of the contracts are specified, then the rest of them are treated as disabled and won't be deployed. So in the example config we'd better specify the whole list of native update histories with proper `[0]` value and add a note about it. Technically, current example config says: "deploy the only GasToken contract at the genezis " which is not the desired behaviour.

Reply:
> treated as disabled and won't be deployed

Is it useful for Inner Ring? IMO it's pretty easy to forget some contract. What do you think? @AnnaShaleva @roman-khimov

At first glance, it seems that it is more convenient to configure custom heights partially, and default to `[0]` for the rest. But maybe I don't fully feel this mechanism.

Reply:
> Is it useful for Inner Ring

It is not used anywhere even inside the Neo networks (at least as far as I know). Even with Notary contract there's a separate `P2PSigExtensions` that perfectly handles the default (missing) NativeActivations.

> it's pretty easy to forget some contract

Just leave this section completely emty then.

> it is more convenient to configure custom heights partially

Or do this logic in ir's config handler. We can't do this in neo-go because it's just the way how we specify disable contract.

Reply:
> Just leave this section completely emty then.

Sad, but I can't. Completely hide or completely clarify, no other options.

> Or do this logic in ir's config handler.

This replaces user error to IR dev's error, so finally I decided to just clarify your initial comment with a loud caution.